pydcs_extensions/frenchpack/frenchpack.py

```diff
@@ -49,7 +49,7 @@ class VAB_T20_13(unittype.VehicleType):
 
 class VAB_MEPHISTO(unittype.VehicleType):
     id = "VAB_HOT"
-    name = "VAB MEPHISTO"
+    name = "ATGM VAB MEPHISTO"
```

Comment:
This is a generated file. Did the mod update?

Reply:
No that was an oversight from me when I tested something. I reverted it.